pkg/common/utils/pointers.go

```diff
@@ -0,0 +1,5 @@
+package utils
+
+func PtrTo[T any](v T) *T {
```

Comment:
We have a `StringPtr` util here:

https://github.com/safedep/vet/blob/main/pkg/common/utils/string.go#L4

`PtrTo` seems more generic. Can we get rid of `StringPtr` to and use this instead? Remove `utils/string.go` if not required.

Reply:
cool, will replace with `PtrTo`